packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverBody/useTeachingPopoverBodyStyles.styles.ts

```diff
@@ -1,33 +1,62 @@
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { SlotClassNames } from '@fluentui/react-utilities';
 import type { TeachingPopoverBodySlots, TeachingPopoverBodyState } from './TeachingPopoverBody.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const teachingPopoverBodyClassNames: SlotClassNames<TeachingPopoverBodySlots> = {
   root: 'fui-TeachingPopoverBody',
-  // TODO: add class names for all slots on TeachingPopoverBodySlots.
-  // Should be of the form `<slotName>: 'fui-TeachingPopoverBody__<slotName>`
+  media: 'fui-TeachingPopoverBody__media',
 };
 
-/**
- * Styles for the root slot
- */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'flex',
+    flexDirection: 'column',
+    paddingBottom: '12px',
   },
-
-  // TODO add additional classes for different states and/or slots
+  media: {
+    width: '288px',
+    objectFit: 'cover',
+    marginBottom: '12px',
+    verticalAlign: 'middle',
+  },
+  mediaShort: { height: '117px' },
+  mediaMedium: { height: '176px' },
+  mediaTall: { height: '288px' },
 });
 
-/**
- * Apply styling to the TeachingPopoverBody slots based on the state
- */
-export const useTeachingPopoverBodyStyles_unstable = (state: TeachingPopoverBodyState): TeachingPopoverBodyState => {
+/** Applies style classnames to slots */
+export const useTeachingPopoverBodyStyles_unstable = (state: TeachingPopoverBodyState) => {
+  const { mediaLength } = state;
   const styles = useStyles();
+
+  let mediaHeightStyle = styles.mediaMedium;
+  switch (mediaLength) {
```

Comment:
A pattern we use elsewhere in Fluent for this typeo of thing is to define the style keys to match the possible values of `mediaLength`, this way you can just write `const mediaHeightStyle = styles[mediaLength]`.

Ex: https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.styles.ts#L488